File: src/main/java/site/gachontable/gachontablebe/domain/shared/exception/PasswordNotMatchException.java
package site.gachontable.gachontablebe.domain.shared.exception;


import site.gachontable.gachontablebe.global.error.ErrorCode;
import site.gachontable.gachontablebe.global.error.exception.ServiceException;

public class PasswordNotMatchException extends ServiceException {
    public PasswordNotMatchException() {
        super(ErrorCode.PASSWORD_NOT_MATCH);
    }
}

File: src/main/java/site/gachontable/presentation/shared/exception/PasswordNotMatchException.kt
package site.gachontable.presentation.shared.exception;


import site.gachontable.independent.type.ErrorCode;
import site.gachontable.independent.error.exception.ServiceException;

public class PasswordNotMatchException extends ServiceException {
    public PasswordNotMatchException() {
        super(ErrorCode.PASSWORD_NOT_MATCH);
    }
}
